Doc/library/argparse.rst

```diff
@@ -1745,10 +1745,12 @@ Sub-commands
      encountered at the command line
 
    * dest_ - name of the attribute under which sub-command name will be
-     stored; by default ``None`` and no value is stored
+     stored; by default ``None`` and no value is stored. Required if
+     required_ has been passed
```

Comment:
‘Required if required’ doesn’t read well. 

maybe just remove this addition. Is it not enough to document this under ‘required’?

Reply:
Apologies for the radio silence, I have removed the line as requested, as yes it isn't needed